nl.json

```diff
@@ -1,6 +1,5 @@
 {
-    "name" : "Nederlands",
-    "name-en" : "Dutch",
+    "name" : "Dutch",
```

Comment:
`name` should be the name of the language in this language. However, this change is not necessary. See above.